converter/dicom-cast/src/Microsoft.Health.DicomCast.Hosting/appsettings.json

```diff
@@ -37,6 +37,9 @@
         "PollInterval": "00:01:00",
         "PollIntervalDuringCatchup": "00:00:00"
     },
+    "DicomValidation": {
+        "PartialValidation" : false
```

Comment:
Why is it false here and true in development json

Reply:
I removed it from development json and left it as false here for the default.